R/tm_missing_data.R

```diff
@@ -1197,5 +1208,46 @@ srv_missing_data <- function(id, datasets, dataname, plot_height, plot_width, gg
       datanames = dataname,
       modal_title = "R code for missing data "
     )
+
+    ### REPORTER
+    if (with_reporter) {
+      card_fun <- function(comment) {
+        card <- teal.reporter::TealReportCard$new()
+        summary_type <- input$summary_type
+        title <- paste0(summary_type, " plot")
+        card$set_name(paste0("Missing data - ", title))
+        card$append_text(title, "header2")
```

Comment:
?

Reply:
to be consistent I will use it directly. I leave it to you.

Reply:
I use it twice so I prefer to define it and reuse it.